jwst/datamodels/schemas/slitdata.schema.yaml

```diff
@@ -94,6 +94,12 @@ allOf:
     ycen:
       title: Center of shutter in MSA coordinates (Y dir)
       type: integer
+    source_ra:
+      title: Right Ascention of the source
+      type: number
+    source_dec:
+      title: Declination of the source
+      type: number
```

Comment:
If these are to be saved to the FITS headers we need FITS keywords. Any suggestions?

Reply:
The target coordinates in the primary header have keywords TARG_RA, TARG_DEC.  We could use RA_TARG, DEC_TARG for the keywords in extension headers.  Or perhaps RA_SLIT, DEC_SLIT?

Reply:
Why not just use the existing FITS keywords TARG_RA and TARG_DEC, but put them in the correct SCI extension?  This would cause the least confusion I think.

Reply:
Personally I don't like repurposing existing keyword names for other uses and especially don't want to confuse users by having them appear in both the primary and SCI headers, with different values. A lot of the existing source-related keywords that are populated in the SCI header for multi-slit data use the form "SRCxxxx" (e.g. SRCNAME, SRCTYPE, SRCXPOS, ...) so I suggest we continue that scheme and use "SRCRA" and "SRCDEC".

Reply:
Agree.